istio-telemetry/kiali/templates/clusterrole.yaml

```diff
@@ -0,0 +1,239 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kiali
+  labels:
+    app: {{ template "kiali.name" . }}
```

Comment:
For the new installer we are trying to reduce the use of tiller and helm labels - chart, heritage have been removed from all other templates. 

We keep using release and app - but without the template - just "app: kiali" should be sufficient.

Reply:
sure, updated it